sns-book/src/domain-name/records.md

```diff
@@ -72,13 +76,22 @@ A more generic resolving function `getRecord` is also exported with the followin
 (connection: Connection, domain: string, record: Record) => Promise<NameRegistryState>
 ```
 
+## The SOL record
+
+The SOL record can be used to receive funds to a different address than the one owning the domain. This allows people to hold the domain on a cold wallet while still being able to receive funds in a hot wallet.
+
+The SOL record data contains a 96-byte array that is the concatenation of a public key (32 bytes) and signature (64 bytes). The first 32 bytes represent the public key (`pubkey`) to which funds should be sent and the next 64 bytes are the signature of `pubkey_as_bytes + record_key_as_bytes` signed by the owner of the domain. If the signature is invalid funds **must not** be transfered.
```

Comment:
are the signature of `pubkey_as_bytes + record_key_as_bytes` by the owner of the domain